src/app/core/auth/services/auth.service.ts

```diff
@@ -15,4 +15,19 @@ export class AuthService {
       params: new HttpParams({ fromObject: credentials })
     })
   }
+
+  public register(registerData: RegisterData): Observable<RegisterData> {
+    const { name, email, password } = registerData
```

Comment:
Why destructure an object if you assemble it again into the same object on the next line?